src/functionalTest/java/uk/gov/hmcts/reform/pip/data/management/publication/PublicationTest.java

```diff
@@ -206,6 +209,36 @@ private Artefact uploadNonStrategicFile(String courtId, Sensitivity sensitivity,
         return responseUploadNonStrategicFile.as(Artefact.class);
     }
 
+    @Test
+    void testPublicationEndpointsWithHtmlFileUpload() throws Exception {
```

Comment:
as discussed, renamed this function to specify that this test only upload HTML file to S3 bucket.